engines/bops_config/app/views/bops_config/application_types/show.html.erb

```diff
@@ -67,11 +67,13 @@
           <%= t(".status") %>
         </dt>
         <dd class="govuk-summary-list__value">
-          <%= @application_type.status.humanize %>
+          <span class="govuk-tag <%= "govuk-tag--#{tag_colour(@application_type.status)}" %>">
+            <%= @application_type.status.humanize %>
+          </span>
         </dd>
         <dd class="govuk-summary-list__actions">
           <%= link_to [:edit, @application_type, :status], class: "govuk-link" do %>
-            <%= t(".edit") %><span class="govuk-visually-hidden"> <%= t(".status") %></span>
+            <%= t(".change") %><span class="govuk-visually-hidden"> <%= t(".status") %></span>
```

Comment:
I used 'Edit' as that was what was in the Figma design for 'Live date for staging' which is what we've replaced with 'Status' - could be an oversight on Fede's part or may be intentional so we should check with him.

Reply:
Yeah, I've had a catch-up with Fede and made these changes based on his feedback! I did mention that on the Figma design is `edit` but in some order places `change`. He said that it might be something he forgot to update and we agreed to update it to `change` so we are consistent